prism/0.download_prism.ipynb

```diff
@@ -0,0 +1,141 @@
+    "        \"output_data_dir\": \"primary_screen\",\n",
+    "    },\n",
+    "    \"20237715\": {  # Compound details\n",
+    "        \"output_file\": \"primary-screen-replicate-collapsed-treatment-info.csv\",\n",
+    "        \"output_data_dir\": \"primary_screen\",\n",
+    "    },\n",
+    "    \"20237709\": {  # PRISM readouts replicate collapsed\n",
+    "        \"output_file\": \"primary-screen-replicate-collapsed-logfold-change.csv\",\n",
+    "        \"output_data_dir\": \"primary_screen\",\n",
+    "    },\n",
+    "    # Secondary screen\n",
+    "    \"20237769\": {  # Cell line details\n",
+    "        \"output_file\": \"secondary-screen-cell-line-info.csv\",\n",
+    "        \"output_data_dir\": \"secondary_screen\",\n",
+    "    },\n",
+    "    \"20237763\": {  # Compound details\n",
+    "        \"output_file\": \"secondary-screen-replicate-collapsed-treatment-info.csv\",\n",
+    "        \"output_data_dir\": \"secondary_screen\",\n",
+    "    },\n",
+    "    \"20237757\": {  # PRISM readouts replicate collapsed\n",
+    "        \"output_file\": \"secondary-screen-replicate-collapsed-logfold-change.csv\",\n",
+    "        \"output_data_dir\": \"secondary_screen\",\n",
+    "    },\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "482c8691-3cc6-4a83-9135-f69312d9f5c5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Downloading data/primary_screen/primary-screen-cell-line-info.csv...\n",
+      "Downloading data/primary_screen/primary-screen-replicate-collapsed-treatment-info.csv...\n",
+      "Downloading data/primary_screen/primary-screen-replicate-collapsed-logfold-change.csv...\n",
+      "Downloading data/secondary_screen/secondary-screen-cell-line-info.csv...\n",
+      "Downloading data/secondary_screen/secondary-screen-replicate-collapsed-treatment-info.csv...\n",
+      "Downloading data/secondary_screen/secondary-screen-replicate-collapsed-logfold-change.csv...\n"
+     ]
+    }
+   ],
+   "source": [
+    "for figshare_id in download_dict:\n",
+    "    # Create directories and paths for outputing data\n",
+    "    output_dir = pathlib.Path(\n",
+    "        base_output_dir, download_dict[figshare_id][\"output_data_dir\"]\n",
+    "    )\n",
+    "    output_dir.mkdir(parents=True, exist_ok=True)\n",
+    "\n",
+    "    output_file = pathlib.Path(output_dir, download_dict[figshare_id][\"output_file\"])\n",
+    "\n",
+    "    # Download data from figshare\n",
+    "    print(f\"Downloading {output_file}...\")\n",
+    "\n",
+    "    download_utils.download_figshare(\n",
+    "        figshare_id=figshare_id, output_file=output_file, figshare_url=figshare_url\n",
+    "    )"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
```

Comment:
Is this environment / kernel accurate for this notebook? If not, consider rerunning in the correct environment.

Reply:
oof, no. Thanks for pointing this out. Will rerun